### DISCUSSION TITLE
Chronos-Bolt Context length

### DISCUSSION THREAD

**Discussion by gknair** [amazon-science/chronos-forecasting#230] [Q&A] [answered]
The context window of Chronos-Bolt is a max of 2048 , while using AutoGluon predict function, if we provide a timeseries of more than 2048 data points, Does the function trim the series to the last 2048 points before forecasting?

**Comment by lostella** [comment, marked as answer]
@gknair yes, this happens here https://github.com/autogluon/autogluon/blob/bfbfc593898e8bffa6486bb7bcd5a04039d73b0d/timeseries/src/autogluon/timeseries/models/chronos/pipeline/chronos_bolt.py#L417-L418
